### ISSUE TITLE
Setting header: 'Content-Type': 'application/json' is not working

### ISSUE THREAD

**Issue by Ruslan101** [axios/axios#4939]
#### Describe the bug
Unable to pass Content-Type header to get request

#### To Reproduce
```js
    const axiosRequest = axios.create({
      baseURL: 'https://some-domain.com/api',
      timeout: 60000,
    })

    axiosRequest.get('/method/get_some_data', { data: null, headers: { 'Content-Type': 'image/png' } })
    // Next, the request leaves with header parameters content-type: text/html
```

#### Expected behavior
The request is expected to leave with 'Content-Type': 'image/png' header parameters

#### Environment
 - Axios Version [e.g. 0.21.1]
 - Adapter [e.g. XHR/HTTP]
 - Browser [e.g. Chrome]
 - Browser Version [e.g. 104.0.5112.79 ]
 - Node.js Version [e.g. 14.19.3]
 - OS: [e.g. Pop!_OS 22.04 LTS x86_64]

#### Additional context/Screenshots
![image](https://user-images.githubusercontent.com/40547043/188170709-40ababce-b81f-4cc7-92a7-9f90392026ed.png)

**Comment by iambenkay** [comment]
The section in your attachment is a response header not a request header. Kindly check request header to confirm.

**Comment by Ruslan101** [comment]
Yes, the issue has been resolved, this issue is a duplicate: [issues#86](https://github.com/axios/axios/issues/86)
